controllers/tenant/namespaces.go

```diff
@@ -73,6 +73,12 @@ func (r *Manager) syncNamespaceMetadata(ctx context.Context, namespace string, t
 				}
 			}
 
+			if tnt.Spec.Cordoned {
+				labels["projectcapsule.dev/cordoned"] = "true"
+			} else {
+				labels["projectcapsule.dev/cordoned"] = "false"
+			}
+
```

Comment:
We don't want a label, when it's not cordoned. Also with only this, you will cause clashes with gitops engines, as they will also try to patch this away. You should additionally verify at admission, if this label must be set, if patch.

you can add a new handler here:
https://github.com/projectcapsule/capsule/tree/main/pkg/webhook/ownerreference